check_run/overrides/payment_entry.py

```diff
@@ -53,6 +57,144 @@ def get_valid_reference_doctypes(self):
 		elif self.party_type == "Employee":
 			return ("Journal Entry", "Expense Claim")  # Expense Claim
 
+	# validation msg improvement

```

Comment:
Because we have copy both of these function in order to improve the error message, it's probably good to have a little bit longer note here about why. 

```python
"""
Because Check Run processes multiple payment entries in a background queue, errors generally do not include
enough data to identify the problem since there were written and remain appropriate for the context of an individual
Payment Entry. This code is copied from: 

https://github.com/frappe/erpnext/blob/version-14/erpnext/accounts/doctype/payment_entry/payment_entry.py#L164

https://github.com/frappe/erpnext/blob/version-14/erpnext/accounts/doctype/payment_entry/payment_entry.py#L194
"""
```

Reply:
I have added a comment as you suggest